HTML+CSS+JS/Cards/AnimatedCardStacksAnimation/README.md

```diff
@@ -0,0 +1,9 @@
+# Animated Card stack
+
```

Comment:
You need to add a demo GIF or video for your component, take reference from this-
https://github.com/ALPHAVIO/Mirage-UI/tree/master/React/Navigation-Bar/TopDownAnimation-ResponsiveNavbar